eva-accession-import/src/main/resources/application.properties

```diff
@@ -1,38 +1,34 @@
-spring.batch.job.names=IMPORT_DBSNP_VARIANTS_JOB
-
-parameters.assemblyAccession=
-parameters.assemblyName=
-parameters.assemblyReportUrl=
-parameters.taxonomyAccession=
-parameters.chunkSize=100
-#parameters.pageSize=
+spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
+
+# dbsnp
+parameters.dbsnp=/Users/rohnkum/casesafe/acsn/refsnp-sample.json
```

Comment:
No paths in properties files please. Also, we normally name the input file `parameters.input` in other pipelines.